yang/ietf-ucl-acl.yang

```diff
@@ -44,14 +44,72 @@ module ietf-ucl-acl {
       "RFC XXXX: A Policy-based Network Access Control";
   }
 
-  feature match-on-endpoint-group {
+  identity group-acl-type {
+    if-feature "group";
+    base acl:acl-base;
+    description
+      "An ACL that matches based on an abstract group identity,
+       which may be within the NVO3 header extension, but may not
+       correspond to any field in the packet header.";
+  }
+  identity mixed-ipv4-group-type {
+    if-feature "mixed-ipv4-group";
+    base acl:ipv4-acl-type;
+    base uacl:group-acl-type;
+    description
+      "An ACL that contains a mix of entries that match on fields
+       in IPv4 headers and abstract group identities.  Matching on 
```

Comment:
"abstract group identities" is too abstract, IMO.

Reply:
how about this: s/abstract group identities/endpoint group identities/?

Reply:
That is better, but still the description should be updated to call out what is meant by "endpoint group identities"; otherwise it is not that easy for readers to get the intent. Thanks.

Reply:
Updated with: An ACL that matches based on an endpoint group identity,
       which can represent the collective identity of a group of 
       authenticated users or enterprise end devices.
Would this be okay?

Reply:
That's much more better, @QiufangMa. Thanks